packages/driver/src/cy/net-stubbing/add-command.ts

```diff
@@ -129,6 +130,10 @@ function validateRouteMatcherOptions (routeMatcher: RouteMatcherOptions): { isVa
     }
   }
 
+  if (_.isString(routeMatcher.hostname) && !isValidDomain(routeMatcher.hostname, { allowUnicode: true })) {
+    return err('`hostname` must be a valid domain name.')
```

Comment:
I see that this is accurate, but what? 😅 Why did we call this arg hostname if it's really domain name?

Reply:
`hostname` is the appropriate `URL`/`Location` property for this part of the URL - `scheme://hostname:port/path`

`hostname` is a misnomer though, a hostname has a stricter set of validation criteria than an actual domain name that can go in a URL. For example, as I understand it, UTF-8 is not allowed in a hostname, whereas it would be valid in a domain name: https://stackoverflow.com/a/106223/3474615

Reply:
Yeah, just reads a little weird I guess.

Reply:
Agreed, open to suggestions...

```suggestion
    return err('`hostname` must be a valid host name or domain name.')
```